orga/tests/integration/components/campaign/update-form_test.js

```diff
@@ -52,21 +38,48 @@ module('Integration | Component | Campaign::UpdateForm', function (hooks) {
       assert.dom(screen.getByLabelText('Titre du parcours')).hasAttribute('maxLength', '50');
     });
 
+    test('it should contain inputs, attributes, information block,validation and cancel buttons', async function (assert) {
+      // when
+      const screen = await renderScreen(
+        hbs`<Campaign::UpdateForm @campaign={{this.campaign}} @onSubmit={{this.updateCampaignSpy}} @onCancel={{this.cancelSpy}} />`
+      );
+
+      // then
+      assert.dom(screen.getByLabelText('* Nom de la campagne')).exists();
+      assert.dom(screen.getByLabelText('* Propriétaire de la campagne')).exists();
+      assert.dom(screen.getByText('Propriétaire de la campagne', { selector: 'span' })).exists();
+      assert.dom(screen.getByLabelText("Texte de la page d'accueil")).exists();
+      assert.dom(screen.getByLabelText('Titre du parcours')).exists();
+      assert.dom(screen.getByLabelText("Texte de la page d'accueil")).hasAttribute('maxLength', '5000');
+      assert.dom(screen.getByText('Modifier')).exists();
+      assert.dom(screen.getByText('Annuler')).exists();
+    });
+
     test('it should send campaign update action when submitted', async function (assert) {
```

Comment:
**suggestion** : le test ne vérifie pas ce qui est dit dans le `it` : on devrait vérifier que le `updateCampaignSpy` est appelé

Reply:
done